staging/src/k8s.io/client-go/rest/config_test.go

```diff
@@ -139,13 +139,14 @@ func TestDefaultKubernetesUserAgent(t *testing.T) {
 }
 
 func TestRESTClientRequires(t *testing.T) {
-	if _, err := RESTClientFor(&Config{Host: "127.0.0.1", ContentConfig: ContentConfig{NegotiatedSerializer: api.Codecs}}); err == nil {
+	gvCopy := v1.SchemeGroupVersion
```

Comment:
if this is just about making the following lines shorter, I suggest calling this `sgv`. If it's actually about making a copy because it might get mutated, I'd make a function so that each RESTClientFor line gets its own copy.